ecommerce/api.py

```diff
@@ -204,3 +209,38 @@ def get_new_order_by_reference_number(reference_number):
         return Order.objects.get(id=order_id, status=Order.CREATED)
     except Order.DoesNotExist:
         raise EcommerceException("Order {} is expected to have status 'created'".format(order_id))
+
+
+def enroll_user(order):
+    """
+    Enroll user after they made a successful purchase.
+
+    Args:
+        order (Order): An order to be fulfilled
+
+    Returns:
+         None
+    """
+    user_social = order.user.social_auth.get(provider=EdxOrgOAuth2.name)
+    enrollments_client = EdxApi(user_social.extra_data, settings.EDXORG_BASE_URL).enrollments
+
+    exceptions = []
+    enrollments = {}
+    for line in order.line_set.all():
```

Comment:
Theoretically an order can contain multiple course runs so I thought it was best to handle the case. But yes, users should not be able to purchase more than one run at a time